### DISCUSSION TITLE
Foundry Fundamentals Section 2 / Lesson 9 - -m flag issue with forge test

### DISCUSSION THREAD

**Discussion by 0x1057** [Cyfrin/foundry-full-course-cu#2773] [Q&A] [answered]
Per screenshot, getting an error when trying to forge test with the -m flag.  Searched for fix but unable to find.  Also searched forge test --help and did not see a -m flag.  Maybe it was changed?
![Screenshot 2024-10-29 121718](https://github.com/user-attachments/assets/a234ebec-ceca-43f7-8032-52040b3c1a71)

**Comment by terfasquion** [comment]
There's an update below the video regarding what you mentioned. 
You probably missed it :)

**Reply by 0x1057** [reply, marked as answer]
I absolutely did!  I went back to it a while ago and saw it, came here and saw two kind souls had helped my bumbling self along! Thank you so much regardless!

**Comment by EngrPips** [comment]
Hello @nightfallxyz, Please use `forge test --mt testPriceFeedVersionIsAccurate -vvvv` instead

**Reply by 0x1057** [reply]
Thank you so much for helping me out with this! I saw the update and came back here to post as much and saw you two had got to it already.  Excellent work :)

**Reply by EngrPips** [reply]
You are welcome, Keep crushing it.

**Reply by vicjuma** [reply]
I also came about the same. I had not seen the update section and so I had to go the long way ```forge test --help``` and saw the option ```--match-test```. Maybe it could help you next time when you meet any other deprecated flag. The update is amazing. Cyfrin is the best👍
